docs/package.json

```
@@ -109,7 +109,7 @@
     "react-virtualized": "^9.21.1",
     "react-window": "^1.8.5",
     "recast": "^0.20.2",
-    "recharts": "^1.6.2",
+    "recharts": "^2.0.3",
```

Oops I don't think this change was intended.
```suggestion
    "recharts": "^1.6.2",
```